ansible/playbook.yml

```diff
@@ -150,6 +153,23 @@
       shell:
         cmd: "for fl in /usr/lib/postgresql/bin/* ; do ln -sf $fl /usr/bin/$(basename $fl) ; done"
 
+    - name: Enable pgTAP extension
+      become: yes
```

Comment:
`become: yes` should be unnecessary given it's configured for the entire playbook?